Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -952,6 +979,89 @@ def clear_user_sessions_command(client, args):
     return readable_output, {}, raw_response
 
 
+def get_zone_command(client, args):
+    raw_response = client.get_zone(args, args.get('zoneID', ''))
+    if not raw_response:
+        return 'No zones found.', {}, raw_response
+    zones = json.loads(raw_response)
+    readable_output = tableToMarkdown('Okta Zones', zones, headers=[
+                                      'name', 'id', 'gateways', 'status', 'system', 'lastUpdated', 'created'])
+    outputs = {
+        'Okta.Zone(val.id && val.id === obj.id)': createContext(zones)
+    }
+    return (
+        readable_output,
+        outputs,
+        zones
+    )
+
+
+def list_zones_command(client, args):
+    raw_response = client.list_zones()
+    if not raw_response:
+        return 'No zones found.', {}, raw_response
+    zones = json.loads(raw_response)
+    readable_output = tableToMarkdown('Okta Zones', zones, headers=[
+                                      'name', 'id', 'gateways', 'status', 'system', 'lastUpdated', 'created'])
+    outputs = {
+        'Okta.Zone(val.id && val.id === obj.id)': createContext(zones)
+    }
+    return (
+        readable_output,
+        outputs,
+        zones
+    )
+
+
+def apply_zone_updates(zoneObject, zoneName, gatewayIPs, proxyIPs):
+    # If user provided a new zone name - set it
+    if zoneName:
+        zoneObject["name"] = zoneName
+
+    # Set IPs in CIDR mode. Single IPs will be added as /32.
+    if gatewayIPs:
+        CIDRs = [f"{ip}/32" if '/' not in ip else f'{ip}' for ip in gatewayIPs]
+        zoneObject["gateways"] = [{"type": "CIDR", "value": cidr} for cidr in CIDRs]
+
+    if proxyIPs:
+        CIDRs = [f"{ip}/32" if '/' not in ip else f'{ip}' for ip in proxyIPs]
+        zoneObject["proxies"] = [{"type": "CIDR", "value": cidr} for cidr in CIDRs]
+
+    return zoneObject
+
+
+def update_zone_command(client, args):
+
+    if not args.get('zoneName', '') and not args.get('gatewayIPs', '') and not args.get('proxyIPs', ''):
+        return (
+            'Nothing to update',
+            {},
+            'Nothing to update'
+        )
+    zoneID = args.get('zoneID', '')
+    zoneObject = json.loads(client.get_zone(args, zoneID))
```

Comment:
validate if zone has found / returned. If the response is valid continue, otherwise fail